pkg/opavalidation/opavalidation.go

```diff
@@ -15,34 +15,69 @@ import (
 	"github.com/open-policy-agent/opa/rego"
 	"github.com/open-policy-agent/opa/topdown"
 	"github.com/open-policy-agent/opa/types"
+	"github.com/sirupsen/logrus"
 )
 
-// Run is a ValidateRego() wrapper that prints resulting actionItems. This is
+const (
+	DefaultKubeObjectNamespace = "notset" // The namespace to use if one is unspecified
```

Comment:
interesting, I would think default if not set it should be empty

Reply:
I used a default because `ResourceNamespace` is a required field for an Insights actionItem, and I'm emulating the in-cluster behavior of an object always having a namespace (where that object was deployed).